app/services/diaper_partner_client.rb

```diff
@@ -29,6 +29,7 @@ def self.add(attributes, invitation_message)
   def self.get(attributes)
```

Comment:
I am a little in favor of adding keyword arguments so you'd call it like this:
```
.get({id: 200}, query_params: {...})
```
The reason why I prefer that because the method definition tells you more about what the method expects. Thoughts?

Reply:
This is mainly preference, so I am okay with either way :)

Reply:
Done.